tests/src/Traits/WysiwygTrait.php

```diff
@@ -53,38 +53,44 @@ public function pressWysiwygButton(string $field, string $button, ?TraversableEl
    * @param string $text
    *   The text to enter in the textarea.
    * @param \Behat\Mink\Element\TraversableElement|null $region
-   *   (optional) Limit the region to the specific element. Defaults to the full
-   *   page.
+   *   Optional region where the editor is expected to be located. Defaults to
```

Comment:
I think (as Claudiu has said before) that after https://www.drupal.org/project/drupal/issues/711918, the proper way to document an optional parameter is by using the `|null` in the type definition and the `(optional)` in the beginning of the description.